lib/routes/sdu/qd/xyb.ts

```diff
@@ -0,0 +1,87 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import { finishArticleItem } from '@/utils/wechat-mp';
+
+const host = 'https://xyb.qd.sdu.edu.cn/';
+
+const typeMap = {
+    gztz: {
+        title: '工作通知',
+        url: 'gztz.htm',
+    },
+};
+
+export const route: Route = {
+    path: '/qd/xyb/:type?',
+    categories: ['university'],
+    example: '/sdu/qd/xyb/gztz',
+    parameters: { type: '默认为`gztz`' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '山东大学（青岛）学研办',
+    maintainers: ['kukeya'],
+    handler,
+    description: `| 工作通知 | 
+  | -------- |
+  | gztz     | `,
+};
+
+async function handler(ctx) {
+    const type = ctx.req.param('type') ? ctx.req.param('type') : 'gztz';
+    const link = new URL(typeMap[type].url, host).href;
+
+    const response = await got(link);
+
+    const $ = load(response.data);
+
+    let item = $('.list li')
+        .map((_, e) => {
+            e = $(e);
+            const a = e.find('a');
+            return {
+                title: a.text().trim(),
```

Comment:
Please remove the `•` from title in `/sdu/qd/xyb/gztz`
https://github.com/DIYgod/RSSHub/pull/16811#issuecomment-2360623956